tests/sandbox/create_message_batches/test_performance.py

```diff
@@ -36,7 +44,17 @@ def test_create_messages_large_valid_payload(nhsd_apim_proxy_url):
 @pytest.mark.sandboxtest
 def test_create_messages_large_invalid_payload(nhsd_apim_proxy_url):
     """
-    .. py:function:: Test large (50k) invalid payload
+    .. py:function:: Scenario: A user submitting a request with a \
```

Comment:
Do we think `fifty thousand` is easier to read than `50k` or `50,000`?

Reply:
50k didn't read great to me, reminded me 50kb, though I'm not overly partial to fifty thousand, went with 50,000 instead